client/css/style.css

```diff
@@ -1614,8 +1624,8 @@ kbd {
 	animation-fill-mode: forwards;
 	-webkit-animation-timing-function: ease-in;
 	animation-timing-function: ease-in;
-	-webkit-animation-delay: .4s;
-	animation-delay: .4s;
+	-webkit-animation-delay: .1s;
+	animation-delay: .1s;
```

Comment:
Could you revert this please? We actually re-added the animation delay some time ago on purpose to avoid fidgeting on timestamps. We'd be better off with default values.